src/credentials/CertificationDeclaration.h

```diff
@@ -62,6 +63,38 @@ struct CertificationElements
     bool DACOriginVIDandPIDPresent;
 };
 
+struct CertificationDeclarationContent
```

Comment:
This struct is almost identical to the previous one - so maybe to eliminate confusion give it a similar name e.g.: `CertificationElementsWithoutPIDs`

Reply:
Addressed in b5f45aa